Sources/MapboxNavigation/TopBannerViewController.swift

```diff
@@ -54,10 +56,10 @@ open class TopBannerViewController: UIViewController {
     private let instructionsBannerHeight: CGFloat = 100.0
     
     private var informationChildren: [UIView] {
-        return [instructionsBannerView] + secondaryChildren
+        return [instructionsBannerView] + secondaryChildren + [lanesView]
```

Comment:
changes in this file cause LanesView to not show after resuming (after entering preview mode). need to fix